bundles/org.openhab.binding.xmltv/README.md

```diff
@@ -0,0 +1,87 @@
+# XmlTV Binding
+
+XMLTV is an XML based file format for describing TV listings. This format is often used by Home Theater
+software to produce their Electronic Program Guide (http://wiki.xmltv.org/index.php/Main_Page)
+
+The norm allows to separate program display from its building. The building of the XMLTV file itself is
```

Comment:
Can you start each sentence on a new line so it's easier to review changes in the future?